apps/public-docsite-v9/src/DocsComponents/FluentDocsContainer.stories.tsx

```diff
@@ -14,6 +14,9 @@ interface IFluentDocsContainerProps {
 export const FluentDocsContainer: React.FC<IFluentDocsContainerProps> = ({ children, context }) => {
   // eslint-disable-next-line deprecation/deprecation
   const selectedTheme = themes.find(theme => theme.id === context.globals[THEME_ID]);
+  if (context.id === 'theme-theme-designer--page') {
```

Comment:
this code looks brittle, what is the logic behind this ? can it be implemented in some better way? can we provide docs ?